better_exceptions/formatter.py

```diff
@@ -34,6 +34,13 @@
 
 MAX_LENGTH = 128
 
+if os.environ.get("BETTER_EXCEPTIONS_HYPERLINKS") == "1":
```

Comment:
Can this be changed to `!= "0"`?

Reply:
Two problems with that:

1. `.get` will usually return `None`
2. Environment variables can be set to an empty string, which should probably also be considered 'off'.

```suggestion
if os.environ.get("BETTER_EXCEPTIONS_HYPERLINKS", "") not in ("", "0"):
```

Reply:
Cheers, I've updated this following the suggestion by @alexmojaki